index.html

```diff
@@ -0,0 +1,150 @@
+                <h1 class="popheading">Facebook 360</h1>
+                <p class="popp13">FACEBOOK * <span class="popp13">Full Stack Dev</span> * <span class="popp13">2015</span></p>
+                <br>
+                <p class="popp15">A daily selection of privately personalized reads; no accounts <br> or sign-ups required.</p>
+                <span class="htcj3">HTML</span> 
+                <span class="htcj3">CSS</span>
+                <span class="htcj3">Javascript</span><br><br>
+                  <div class="btn3">
+                    <a href="image3.png" target="_blank">See Project</a>
+                  </div>
+        </div>    
+       </div>
+       <div class="container4">
+       
+        <div class="uber">
+                <h1 class="popheading">Uber Navigation</h1>
+                <p class="popp13">Uber Navigation * <span class="popp13">Lead Developer</span> * <span class="popp13">2018</span></p>
+                <br>
+                <p class="popp15">A smart assistant to make driving more safe, efficient, and fun <br> by unlocking 
+                    your most expensive computer: your car..</p><br>
+                <span class="htcj4">HTML</span> 
+                <span class="htcj4">Ruby on Rails</span>
+                <span class="htcj4">CSS</span>
+                <span class="htcj4">Javascript</span><br><br>
+                  <div class="btn4">
+                    <a href="image4.png" target="_blank">See Project</a>
+                  </div> 
+                  </div>
+                  <div class="imgdiv4">
+                    <img src="image4.png" alt="Nature" height="300" width="300">
+                </div>
+        </div>    
+       </div>  
+       <div class="container5">
+        <div class="about">
+            <h1 class="abouth">About Myself</h1>
+            <p class="popp15">Hello I’m a software developer! I can help you build a <br> product , feature or 
+                website Look through some of my <br> work and experience! If you like what you see and
+                 have <br> a project you need coded, don’t hestiate to contact me.</p>
+             <span class="popp15">Let's Connect</span>   
+             <section><i class="fa-brands fa-twitter"></i>
+              <i class="fa-brands fa-linkedin-in"></i>
+              <i class="fa-solid fa-m"></i>
+              <i class="fa-brands fa-github"></i>
+              <i class="fa-regular fa-hand-peace"></i></section><br><br>
+                     <span><a class="abouts" href="#">Get My Resume</a> </span> 
+        </div>
+        <div class="language">
+            <p class="aboutp">Languages</p>
+            <img src="js.png" alt="Js" width="100px" height="100px">
+            <img src="ht.png" alt="Js" width="100px" height="100px">
+            <img src="css.png" alt="Js" width="100px" height="100px">
+            <p class="aboutp">Frameworks</p>
+            <p class="aboutp">Skills</p>
+        </div>
+       </div>
```

Comment:
It is a project requirement that you have major sections, the work section and the about section🤗 Kindly wrap all your projects in a `<section>`  and your  contents about you in a `<section>`🤗 also, your major sections should be aligned with css grid while it's content should be aligned with flexbox😉